system/integrations/pgp/index.rst

```diff
@@ -0,0 +1,111 @@
+PGP
+======
+
+
+Introduction
+------------
+
+PGP is another method for secure email communication.
+With PGP, you can exchange **signed** and **encrypted** messages with others.
+
+Signing
+   is a proof that a message hasn’t been manipulated on it's way.
+
+   In other words, it guarantees a message’s **integrity** and **authenticity**.
+
+Encryption
+   scrambles a message so that it can only be unscrambled by the intended recipient.
+
+   In other words, it guarantees **privacy** and **data security**.
+
+.. figure:: /images/system/integrations/pgp/pgp_ticket_creation.png
+   :alt: Screenshot of ticket creation with encrypt and sign buttons
+   :align: center
+
+   Once PGP has been enabled, 🔒 **Encrypt** and ✅ **Sign** buttons will appear in the ticket composer.
+
+.. note:: **Sign button not visible?**
+
+   Please note that the signing of emails is based on the outgoing email account. 
+   That means you have to choose a group with a sender email account, which has a private key assigned.
+
+Handling of keys
+-----------------
+To use the pgp function, you fist have to enable the integration (PGP). 
+When you swith the toggle to *enabled*, you can add keys by clicking the *add key* button. 
```

Comment:
`swith` => `switch`

The second part of the sentence is not completely correct: admins can manage keys *without* enabling the feature first. Actually, sometime this is preferred :) Perhaps we can simply avoid making connection to the switch, and separate these into two sentences instead?